apps/full-site-editing/full-site-editing-plugin/starter-page-templates/index.js

```diff
@@ -43,6 +44,7 @@ const templatesPluginSharedProps = {
 
 // Open plugin only if we are creating new page.
 if ( screenAction === 'add' ) {
+	dispatch( 'automattic/spt' ).setIsOpen( true );
```

Comment:
nit:  I wonder if we should use `starter-page-templates` instead of `spt` here in the store name to be more verbose.

Reply:
good call! i used `starter-page-layouts` instead, but it's basically the same meaning :p